tst/regression/test_suites/advection_outflow/advection_outflow.py

```diff
@@ -40,7 +40,6 @@ def Analyse(self, parameters):
 
         delta = compare([
             'outflow.out0.00001.phdf',
-            parameters.parthenon_path + '/tst/regression/gold_standard/outflow.out0.00001.phdf'],
-            check_metadata=False)
+            parameters.parthenon_path + '/tst/regression/gold_standard/outflow.out0.00001.phdf'])
```

Comment:
Why don't we want to check metadata? I assumed setting `check_metadata=False` here was a relic from trying to make the test pass with an outdated gold standard. I've updated the gold standard files and so the test passes now with checking metadata.

Reply:
In the case of this test, we're testing the physical output. So in my mind, checking the metadata makes the test more brittle for no reason. But yes, it is true that it will pass with updated gold files.

Reply:
That's a good point and I agree that this test is about the physics output. One thing to consider, though, is that if the HDF5 metadata changes, we have to update the gold files anyway because the `output` test actually tests the output and so I'd say we don't want to ignore metadata there. We could only avoid a gold update if a PR changes the metadata output of the `advection_outflow` test but not of the `output` test (which is possible, but would seem rare to me). I'd say let's leave metadata check turned on for now and if we run into a PR that only changes metadata for this test and not for the output test (which would require a gold update), then I'd be fine with turning off metadata check for this test.

Reply:
@forrestglines hdf5 for yt was one such example. The `output` test failed because it tested metadata, but the `outflow` test still passed.

Reply:
Yes, but in that case the gold files had to be updated because the `output` test failed. So if one updates the `output` gold files anyway, one might as well also update the `outflow` gold files. I'd we way only revert back to not checking metadata if we run into a change that only changes metadata for the `outflow` test and not the `output` test, because in that case it would save us from having to do a new gold release. But if we have to do a new gold release (because `output` test fails), it will also include new files for `outflow` test and so it's ok to keep checking metadata for `outflow` test. Does this make sense?